packaging/PLATFORM_SUPPORT.md

```diff
@@ -0,0 +1,153 @@
+<!--
+title: "Netdata platform support policy"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/PLATFORM_SUPPORT.md
+-->
+
+# Netdata platform support policy
+
+Netdata defines three tiers of official support: Core, Intermediate, and Community. Each tier defines different
+guarantees for platforms in that tier, as follows:
+
+- **Core**: Platforms in the core support tier are our top priority. They are covered rigorously in our CI, usually
+  include official binary packages, and any platform specific bugs recieve a high priority. From the perspective
+  of our developers, platforms in the core support tier _must_ work, with almost no exceptions.
+- **Intermediate**: Platforms in the intermediate support tier are those which Netdata wants to support, but cannot
+  justify core level support for. They are also covered in CI, but not as rigorously as the core tier, may or may
+  not include official binary packages, and any platform specific bugs recieve a normal priority. This is generally
+  where we will add new platforms we have decided to officially support ourselves.
+- **Community**: Platforms in the community support tier are those which are primarily supported by community
+  contributors. They may recieve some support from Netdata, but are only a best-effort affair. When a community
+  member makes a contribution to add support for a new platform, that platform generally will start in this tier.
+
+Additionally, we define two categories for special cases that we do not support:
+
+- **Third-party Supported Platforms**: Some platform maintainers actively support Netdata on their platforms even
+  though we do not provide official support. Third-party supported platforms may work, but the experience of using
+  Netdata on such platforms is not something we can guarantee. Users reporting bugs when using externally supported
+  platforms will be asked to either reproduce the issue on a supported platform or direct their support request
+  to the platform maintainers.
+- **Previously Supported**: As platforms become End Of Life upstream, Netdata will stop officially supporting
+  them. We will generally not go out of our way to break things on these platforms, but we will also not make any
+  effort to ensure that things keep working on them either. Additionally, Netdata may occasionally move an
```

Comment:
Actually, I entirely agree on this one. I usually try to avoid the more ambiguous idioms like this in documentation for exactly this reason, but am not always successful at remembering to do so.